crates/ink/ir/src/ir/attrs.rs

```diff
@@ -363,6 +373,8 @@ pub enum AttributeArgKind {
     /// `#[ink(selector = _)]`
     /// `#[ink(selector = 0xDEADBEEF)]`
     Selector,
+    /// `#[ink(signature_topic = "DEADBEEF")]`
```

Comment:
This is an invalid `signature_topic`, should be 32 bytes? We could do with a ui test for that.

Reply:
This is just a comment. The string length should be checked. But I agree that it wouldn't hurt adding a UI test for this